code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -523,30 +543,30 @@ nobiliumsuppression = INFINITY
 /datum/gas_reaction/nitrylformation/init_reqs()
 	min_requirements = list(
 		/datum/gas/oxygen = 10,
-		/datum/gas/nitrogen = 20,
+		/datum/gas/nitrogen = 10,
 		/datum/gas/bz = 5,
-		"TEMP" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST * 60
+		"TEMP" = FIRE_MINIMUM_TEMPERATURE_TO_EXIST + 60
```

Comment:
this seems like a massive massive change,  explain?

Reply:
since the gases are going to get used more and more with the current iteration, having a gas (nitryl) that requires over 20000 K to make would discurage the players from making it, thus discouraging them from making the more advanced ones (like stim and cyrion B). Lowering it make it also easier for new player to approach it since you don't need fusion heat or a burning canister full of o2 and plasma to make this work (so just using an heater will work here)

Reply:
450 seems a little low, maybe something a little higher? thoughts?

Reply:
i'll make it higher, 1500 K should be good enough